.github/workflows/ci.yml

```diff
@@ -303,22 +356,22 @@ jobs:
       max-parallel: 4
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python 3.8
-      uses: actions/setup-python@v1
-      with:
-        python-version: 3.8
-        architecture: 'x64'
-    - name: Install dependencies
-      run: pip3 install -r requirements.txt
-    - name: Publish python packages
-      env:
-        PYPI_USERNAME: ${{ secrets.PYPI_USERNAME }}
-        PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-      run: ./publish.sh
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+          architecture: 'x64'
+      - name: Install dependencies
+        run: pip3 install -r requirements.txt
+      - name: Publish python packages
+        env:
+          PYPI_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: ./publish.sh
 
   complete:
-    needs: [webknossos_linux, wkcuber_linux, wkcuber_win, wkcuber_mac, wkcuber_docker, docs]
+    needs: [webknossos_linux, wkcuber_linux, wkcuber_win, wkcuber_mac, wkcuber_docker, docs, cluster_tools]
```

Comment:
Should we also run the webknossos or wkcuber package CIs if we change cluster_tools? We could run them in parallel without waiting for the status of cluster_tools.

Reply:
Yes, good point. I would assume it makes sense to always run the CI for modules when their dependencies change. I.e., if clustertools change webknossos and wkcuber need to run. If webknossos changes, wkcuber needs to run. Do you agree?

While we are at it:
![image](https://user-images.githubusercontent.com/2486553/145426656-26944100-514b-4f35-be48-3e4b025bdd86.png)

Do you know why `wkcuber_{mac,win}` don't consume `changes`? Is this intended? I would have thought they should (together with `webknossos_linux`) consume `changes` and then only run if some of `{wkcuber,webknossos,clustertools}` change.

Reply:
I guess the idea was that wkcuber has to run in any case since it depends on all other packages. But we can still enumerate all, probably nicer to check. Also, currently it depends on webknossos, which is not necessary and just prolongues the run-times. The whole pipeline will fail if any of them fails anyways.